lib/utils.sh

```diff
@@ -96,7 +96,7 @@ print_index_tab(){
   delete_on_exit "$temp_headers_file"
 
   if [ -r "$etag_file" ]; then
-    curl_opts=(--header "If-None-Match: $(cat "$etag_file")")
+    curl_opts=(--location --header "If-None-Match: $(cat "$etag_file")")
   fi
 
   index=$(curl --fail --silent --dump-header "$temp_headers_file" ${curl_opts+"${curl_opts[@]}"}  "${NODEJS_ORG_MIRROR}index.tab")
```

Comment:
Right now the header is only being added if we have some cache already installed. We want always to follow redirects, so we need to add it to the command itself on line 102

```suggestion
    curl_opts=(--header "If-None-Match: $(cat "$etag_file")")
  fi

  index=$(curl --fail --silent --location --dump-header "$temp_headers_file" ${curl_opts+"${curl_opts[@]}"}  "${NODEJS_ORG_MIRROR}index.tab")
```